packages/api/src/routes/medical/document.ts

```diff
@@ -125,10 +126,15 @@ router.post(
     const forceCommonwell = stringToBoolean(getFrom("query").optional("commonwell", req));
     const forceCarequality = stringToBoolean(getFrom("query").optional("carequality", req));
 
+    // TODO ENG-618: Get patients first facility id if no facility id is provided
```

Comment:
The reason for this is because although our docs say its required some customers dont pass a facilityId. So this is Temp until i let customers know that we are going to update that

Reply:
Sounds good. Can we just update the comment to explain why we're doing this, instead of what's being done? The code is clear on what's happening, we just need to explain why and that's a temporary thing.